packages/web/src/components/createCell.tsx

```diff
@@ -319,11 +319,12 @@ export function createCell<
           // so using `any` here
           queryRest = { variables } as any
         } else {
-          queryCache[cacheKey] ||= {
-            query,
-            variables: options.variables,
-            hasProcessed: false,
-          }
+          queryCache[cacheKey] ||
+            (queryCache[cacheKey] = {
+              query,
+              variables: options.variables,
+              hasProcessed: false,
+            })
```

Comment:
Yeah, exactly. Didn't want to spend more time trying to figure out why it was only a problem for me and no one else. So I just rewrote it instead to what it would have been transpiled to